src/test/ui/rfc-2294-if-let-guard/bindings.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(if_let_guard)]
+#![allow(incomplete_features)]
+
+fn main() {
+    match Some(None) {
+        Some(x) if let Some(y) = x => (x, y),
+        _ => y, //~ ERROR cannot find value `y`
+    }
+    y //~ ERROR cannot find value `y`
```

Comment:
Shouldn't this define an `x` and a `y` so this test can be `check-pass`? That way we can be absolutely certain that it works.

Reply:
Oops, I just realized it's intentional that `y`'s not defined ;)